torch/_dynamo/polyfills/pytree.py

```diff
@@ -58,11 +91,10 @@ def _(*args: Any, **kwargs: Any) -> bool:
     "namedtuple_fields",
     "structseq_fields",
 ):
-    __func = getattr(optree, __name)
+    __func = globals()[__name]
```

Comment:
Could we be more specific than `globals()` here? i.e are these from the *tree namespaces?

Reply:
I changed to another approach by using the variable directly.